toolkit/phone-assessment-interviews/personal-relationships.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: article
+permalink: /toolkit/phone-assessment-interviews/personal-relationships/
+section: hiring-phases
+sidenav: hiring-phases
+phase: phone-assessment-interviews
+title: Guidance on Personal Relationships with Applicants
+description: Use this guidance to know whether the SME should recuse themselves from assessing an applicant.
+---
+
+It's common for SMEs participating in a hiring action to know applicants. SMEs may interview people they know, supervise, and work with, such as contract professionals or fellow public servants, except in these cases:
+
+- The SME has a personal relationship with the applicant outside of work, whether or not the relationship is known to others.
+- There is any _appearance_ of bias from the SME toward the applicant, such as when it's well known that the SME and applicant have regular disagreements, or that the SME and the applicant have a unique bond.
+
+In general, if the SME and the applicant have occasionally attended the same office happy hour, there is no conflict. However, if the SME and applicant are regular drinking buddies, the SME should recuse themselves.
+
+### Interviewing an Acquaintance
+
+Phone interview assessments are examinations that applicants must pass based the comparison of their responses to questions and the associated competencies and proficiency levels. If an SME interviews an acquainted applicant during qualifying interviews, they may **only** consider the applicant's interview responses to qualify them. While SMEs may reasonably draw conclusions from an applicant's response to questions, SMEs may not use outside knowledge of the applicant's performance to make a determination.
+
+Hiring managers may choose to conduct additional interviews with applicants who make it to the hiring certificate, and may consider past performance and personal knowledge of how the applicant works with others during those interviews.
```

Comment:
however, any interviews conducted after the certificate in this process are really suitability interviews and not to deem if applicants are qualified.

Reply:
Also, to actually explain the role in detail since that won't have happened yet.